compass/landice/tests/mismipplus/albany_input.yaml

```diff
@@ -3,6 +3,10 @@
 ANONYMOUS:
   Build Type: Tpetra
 
+  Problem:
```

Comment:
@mperego , MISMIP+ requires a specified rate factor of "A=6.338 × 10−25 Pa−3 s−1 Glen’s law coefficient = 2.0 × 10−17 Pa−3 a−1", and it looks like that setting has gotten lost at some point.  Can you instruct us the current way to specify that in the yaml file here?

Reply:
@mperego , also do we need this bit?
https://github.com/MPAS-Dev/MPAS-Model/blob/landice/develop/testing_and_setup/compass/landice/MISMIP%2B/albany_input.yaml#L24-L27

Reply:
> @mperego , MISMIP+ requires a specified rate factor of "A=6.338 × 10−25 Pa−3 s−1 Glen’s law coefficient = 2.0 × 10−17 Pa−3 a−1", and it looks like that setting has gotten lost at some point. Can you instruct us the current way to specify that in the yaml file here?

You should add this lines in the `Problem` section
  ```
Problem: 
    LandIce Viscosity:
      Type: 'Glen''s Law'
      Flow Rate Type: Uniform
      Glen's Law A: 6.338e-25
```
You can avoid the last line if you set the value in MPAS.

Reply:
> @mperego , also do we need this bit? https://github.com/MPAS-Dev/MPAS-Model/blob/landice/develop/testing_and_setup/compass/landice/MISMIP%2B/albany_input.yaml#L24-L27

You don't really need it.